docs/articles/nunit/getting-started/installation.md

```diff
@@ -1,43 +1,149 @@
+---
+uid: installation
+---
+
 # Installation
 
-To get a copy of the NUnit 3, you can use various installation approaches.
+To install NUnit, you can use various installation approaches, depending upon your preferred development environment and preferences.
+
+## I'm using Visual Studio as my development tool
+
+You can add a NUnit project from the Add New Project dialog. Search for NUnit and choose the NUnit Test Project template.
+
+This will add a new project to your solution, containing a single test class and a single test method. Your project file will have the necessary references to NUnit, the NUnit Test Adapter, the NUnit Analyzer, Microsoft Test SDK and the coverlet code coverage package.
+
+Be aware that the templates behind this may be out of date, so always go to the Manage NuGet Packages dialog and update to the latest version of all packages there.
```

Comment:
Going to turn this into an info section. Adding this TODO here for myself within this PR.